### PR TITLE
feat(client|redux): returns - split core and redux, general mocks & return statements

### PR DESCRIPTION
**Pull request by jpklzm**
## Description

This splits the returns client into isolated client and redux actions. It also adds result returns for the actions and moves the fixtures to the proper place in the project.

BREAKING CHANGE
- Clients are now exported from `@blackout-client/returns/*` instead of `@blackout-client/core/returns/client/*`.
- Everything under `@blackout-client/core/returns/redux/*` was moved to `@blackout-redux/returns/*`.
- You should update the client and redux imports with the new `@blackout-client` and `@blackout-redux` packages, respectively.

### Dependencies

None

## Checklist

- [x] The commit message follows our guidelines
- [x] Tests for the respective changes have been added
- [x] The code is commented, particularly in hard-to-understand areas
- [x] The labels and/or milestones were added

## Disclaimer

By sending us your contributions, you are agreeing that your contribution is made subject to the terms of our [Contributor Ownership Statement](https://github.com/Farfetch/.github/blob/master/COS.md)

### CODE REVIEW

File: packages/redux/src/entities/schemas/return.js [hunk 1]
```diff
@@ -0,0 +1,59 @@
+import { adaptDate } from '@farfetch/blackout-client/helpers/adapters';
```
Comment (jhairabreu) [inline packages/redux/src/entities/schemas/return.js:1]: Did you delete this file from the `packages/client/entities`? Since this is a new file and not a moved one, i think you might have missed that. Can you check?

File: packages/redux/src/entities/schemas/returnItem.js [hunk 2]
```diff
@@ -0,0 +1,9 @@
+import { schema } from 'normalizr';
```
Comment (jhairabreu) [inline packages/redux/src/entities/schemas/returnItem.js:1]: Same with this file.

File: yarn.lock [hunk 3]
```diff
@@ -14,7 +14,7 @@
   resolved "https://registry.yarnpkg.com/@babel/compat-data/-/compat-data-7.16.4.tgz#081d6bbc336ec5c2435c6346b2ae1fb98b5ac68e"
   integrity sha512-1o/jo7D+kC9ZjHX5v+EHrdjl3PhxMrLSOTGsOdHJ+KL8HCaEK6ehrVL2RS6oHDZp+L7xLirLrPmQtEng769J/Q==
 
-"@babel/core@^7.1.0", "@babel/core@^7.12.3", "@babel/core@^7.4.4", "@babel/core@^7.7.5":
+"@babel/core@7.13.15", "@babel/core@^7.1.0", "@babel/core@^7.12.3", "@babel/core@^7.4.4", "@babel/core@^7.7.5":
```
Comment (jhairabreu) [inline yarn.lock:17]: Are these changes intentional? Usually we don't need to change these files in the scope of the tasks for this PR

**Review by jhairabreu — CHANGES_REQUESTED** [review]

**Review by jhairabreu — CHANGES_REQUESTED** [review]
You need to mark the "split core and redux" commit as BREAKING CHANGE. Check other PR's where we did this to see how.

**Review by nelsonleite — CHANGES_REQUESTED** [review]
Please review your commit and PR title's scope (in next we have no `core` package) 🙂

**Comment by jpklzm** [comment]
> Please review your commit and PR title's scope (in next we have no `core` package) 🙂

Force of habit. Solving it right now. 🙂

File: yarn.lock (hunk 3 above)
Comment (jpklzm) [reply yarn.lock]: Not intentional, just removed it.

File: packages/redux/src/entities/schemas/return.js (hunk 1 above)
Comment (jpklzm) [reply packages/redux/src/entities/schemas/return.js:1]: I've missed, just solved right now.

File: packages/redux/src/entities/schemas/returnItem.js (hunk 2 above)
Comment (jpklzm) [reply packages/redux/src/entities/schemas/returnItem.js]: Same, missed but already solved 😄

**Comment by jpklzm** [comment]
> You need to mark the "split core and redux" commit as BREAKING CHANGE. Check other PR's where we did this to see how.

Done

File: packages/redux/src/entities/schemas/return.js (hunk 1 above)
Comment (jhairabreu) [reply packages/redux/src/entities/schemas/return.js:1]: 👍

File: packages/redux/src/entities/schemas/returnItem.js (hunk 2 above)
Comment (jhairabreu) [reply packages/redux/src/entities/schemas/returnItem.js]: 👍

**Review by jhairabreu — APPROVED** [review]

**Review by ruifcnunes — APPROVED** [review]

**Review by pedro-gbf — APPROVED** [review]
